extensions/wikia/CommunityPage/scripts/ext.communityPage.js

```diff
@@ -105,19 +128,19 @@ require([
 				}
 			};
 			uiModal.createComponent(createPageModalConfig, function (modal) {
-				getModalTabContentsHtml(tabToActivate).then(function (tabContentHtml) {
-					var html = navHtml + tabContentHtml;
+				var loading = mustache.render(templates.modalLoadingScreen, {
+					loadingText: $.msg('communitypage-modal-tab-loading'),
+				}), html = navHtml + loading;
 
-					modal.$content
-						.addClass('ContributorsModule ContributorsModuleModal')
-						.html(html)
-						.find(tabToActivate.className).children().addClass('active');
+				modal.$content
+					.addClass('ContributorsModule ContributorsModuleModal')
```

Comment:
Class names [should be lower-case-dashed](https://github.com/Wikia/guidelines/blob/master/SCSS/CodingConventions.md#class-and-id-naming-conventions).